.travis.yml

```diff
@@ -1,7 +1,47 @@
 language: php
+
+sudo: false
+
+cache:
+    directories:
+        - $HOME/.composer/cache
+
 php:
-    - 5.5
     - 5.6
     - 7.0
-install: composer install --no-interaction
-script: ./vendor/bin/phpunit
+    - 7.1
+
+env:
+    global:
+        - MOODLE_BRANCH=MOODLE_32_STABLE
+    matrix:
```

Comment:
At the moment, the existing tests don't run in the context of Moodle's PHPUnit test framework and PHPUnit test database. I appreciate that the existing PHPUnits have been written in such a way that a test database isn't currently required, however, it's a Moodle plugin and its tests should really be executable as part of the Moodle PHPUnit test framework.

It's certainly possible that, in the future, test coverage of functionality that touches the database may benefit from being written (e.g. test coverage of the functionality that processes the queue of unsent statements in the `logstore_xapi_log` table). In which case, the test database would end up being necessary anyway.

Finally, my guess (I haven't checked this) is that the Moodlerooms stuff relies on Moodle actually being installed, so my guess is that the above probably does have to be there (I've taken the `.travis.yml` file from [here](https://github.com/moodlerooms/moodle-plugin-ci/blob/master/.travis.dist.yml) so it's very close to the Moodlerooms distribution file).